contributing/code/security.rst

```diff
@@ -103,8 +103,11 @@ Security Advisories
 This section indexes security vulnerabilities that were fixed in Symfony
 releases, starting from Symfony 1.0.0:
 
-* November 23, 2015: `CVE-2015-8125: Potential Remote Timing Attack Vulnerability in Security Remember-Me Service <http://symfony.com/blog/cve-2015-8125-potential-remote-timing-attack-vulnerability-in-security-remember-me-service>`_ (2.3.35, 2.6.12 and 2.7.7)
```

Comment:
Yes! I'm sorry. I replaced `http://` by `https://` in the links of November 2015 (they were the only ones without HTTPS). Too bad the `diff` algorithm used by GitHub doesn't detect that.